src/libstd/panicking.rs

```diff
@@ -180,7 +180,7 @@ fn default_hook(info: &PanicInfo) {
 
     let location = info.location().unwrap();  // The current implementation always returns Some
 
-    let msg = match info.payload().downcast_ref::<&'static str>() {
+    let msg = match info.payload().downcast_ref::<&str>() {
```

Comment:
How come this was necessary? I thought `Any`-style things could only be with `'static` types?